libs/common/src/helpers/fee-record-corrections/map-correction-reasons-to-values.ts

```diff
@@ -0,0 +1,35 @@
+import { FeeRecordEntity } from '../../sql-db-entities';
+import { RECORD_CORRECTION_REASON } from '../../constants';
+
+/**
+ * returns the value of the fee record property that corresponds to the reason
+ * or '-' if the reason is not recognised such as RECORD_CORRECTION_REASON.OTHER
+ * @param feeRecord - the fee record
+ * @param reason - the reason for the correction
+ * @returns - string with the value of the fee record property that corresponds to the reason
+ */
+export const correctionReasonsToValues = (feeRecord: FeeRecordEntity, reason: string) => {
+  switch (reason) {
+    case RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT:
+      return feeRecord.facilityId;
+    case RECORD_CORRECTION_REASON.REPORTED_CURRENCY_INCORRECT:
+      return feeRecord.totalFeesAccruedForThePeriodCurrency;
+    case RECORD_CORRECTION_REASON.REPORTED_FEE_INCORRECT:
+      return feeRecord.totalFeesAccruedForThePeriod;
```

Comment:
I think this should be:
```suggestion
      return feeRecord.feesPaidToUkefForThePeriod;
```

Reply:
changed